frontend/components/users/profile/OrganiserCalendar.tsx

```
@@ -0,0 +1,288 @@
+                  classNames={{
+                    today: ``,
+                    selected: `bg-black border-black text-white rounded-full`,
+                    chevron: `text-black`,
+                  }}
+                />
+              </div>
+            </div>
+
+            {/* Events List View */}
+            <div className="w-1/2 flex-shrink-0">
+              <button onClick={handleBackToCalendar} className="mb-4 flex items-center gap-2 text-sm font-medium">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)